ops/terraform/modules/resources/bfd_monitoring/main.tf

```diff
@@ -0,0 +1,98 @@
+locals {
```

Comment:
General comment - the title and description for this PR need to be a bit more informative as to what we are implementing with this new module.

As I remember it, this was going to be a step toward deprecating the access.txt so this PR provides a new parallel set of metrics and dashboards based on the access.json. Once this PR is deployed and we are confident that it provides the same functionality, we will have a follow on PR for actually converting the existing metrics over to use the json file and we will remove these metrics. Is that recollection correct? If so it would be nice to spell out the whole plan in the PR description.

Reply:
Thanks @christrotteradhoc for feedback, I have detailed the title of this PR as well as the description, please take a look at it.